src/hint_processor/builtin_hint_processor/dict_manager.rs

```diff
@@ -88,6 +90,12 @@ impl DictManager {
             ));
         };
 
+        for val in initial_dict.values_mut() {
+            if val.is_negative() {
+                *val = val.mod_floor(&vm.prime);
+            }
+        }
+
```

Comment:
Is it necessary to mutate the initial_dict we receive? Can't we mutate our copy of the dictionary, as we are already cloning it afterwards

Reply:
Resolved: https://github.com/lambdaclass/cairo-rs/pull/614/commits/9041c0cd93f048da6a252f3ec7aa0742d6321ffe